source/D2Game/src/GAME/SCmd.cpp

```diff
@@ -92,7 +92,7 @@ void __fastcall sub_6FC3C6D0(int32_t nClientId, uint32_t nErrorCode)
 }
 
 //D2Game.0x6FC3C6F0
-void __fastcall D2GAME_PACKETS_SendPacket0x4F_StartMercList_6FC3C6F0(D2ClientStrc* pClient, uint8_t nHeader)
+void __fastcall D2GAME_PACKETS_SendPacket0x4F_UpdateClientState(D2ClientStrc* pClient, uint8_t nHeader)
```

Comment:
Is 0x4f in the name the right thing? I haven't checked in depth but this almost feels like SendOneBytePacket where the last arg is the packet header id. Tho there the other packet ids don't make much sense.

Reply:
this function does indeed send generic single byte packets (mainly used for game join/leave events, but occasionally used for other packets such as the merc list init). `D2GAME_PACKETS_SendPacket0x4F_UpdateClientState` is a misleading name, I'd suggest something along the lines of `D2GAME_PACKETS_SendDatalessPacket` or `D2GAME_PACKETS_SendHeaderOnlyPacket`

Reply:
I'll change it again, I didn't have a look at the actual implementation I think you are correct ! Fixing.